iocore/net/UnixNetAccept.cc

```diff
@@ -425,7 +448,11 @@ NetAccept::acceptFastEvent(int event, void *ep)
   con.sock_type = SOCK_STREAM;
 
   UnixNetVConnection *vc = nullptr;
-  const int loop         = NetAccept::accept_till_done;
+  int count              = 0;
```

Comment:
See this quite a bit - it should be a method or, as noted earlier, the valued move to `NetAccept`.